src/wwwroot/partials/settings/my-plan.html

```diff
@@ -7,21 +7,21 @@
   </div>
   <div class="section--wrapper my-plan--info-container">
     <div class="item flex flex--wrap first">
-      <div class="flex one flex--wrap">
+      <div class="flex one flex--wrap current-plan-price">
```

Comment:
If this is going to be a class for styling something then it's bad formatted, but if this it's only used as an identifier(i think this it's your case) OR a class inside another used for styling, then it's ok.